reactivexcomponent/reactivexcomponent/communication/publisher.py

```diff
@@ -0,0 +1,96 @@
+""" Publisher """
+
+import json
+import websocket
+from reactivexcomponent.configuration.serializer import *
+from lxml import etree
+
+class Publisher:
+	
+	def get_xml_content(self):
+		tree=etree.parse(self.file)
+		tree1=etree.tostring(tree)
+		rt=etree.fromstring(tree1)
```

Comment:
give meaningful names to variables... "root" not "rt"

the same comment for tree1, n, ns, n etc...

refactoring is cheap, use it!